website/docs/components/form/checkbox/partials/code/how-to-use.md

```diff
@@ -238,7 +238,7 @@ Use the standard HTML `checked` attribute to mark the input as checked.
 
 #### Indeterminate
 
-In addition to the checked and unchecked states, a checkbox can be in an indeterminate state, also referred to as partially checked state. A common use case for this state is when a parent checkbox allows the user select multiple children checkboxes at once.
+In addition to the checked and unchecked states, a checkbox can be in an indeterminate state, also referred to as partially checked state. A common use case for this state is when a parent checkbox allows the user select multiple children checkboxes at once. The visual appearance of the checkbox is determined by the component based on the `indeterminate` attribute using JavaScript.
```

Comment:
```suggestion
In addition to the checked and unchecked states, a checkbox can be in an indeterminate state, also referred to as partially checked state. A common use case for this state is when a parent checkbox allows the user to select multiple child checkboxes at once. The visual appearance of the checkbox is determined by the component and based on the `indeterminate` attribute using JavaScript.
```
I'm not 100% sure what you mean by "determined by the component", but I think it needs an "and" in there. Otherwise, I might dropping that part:

"The visual appearance of the checkbox is determined by the `indeterminate` attribute using JavaScript"